computer/linux_server.py

```diff
@@ -69,8 +69,8 @@ def read_file(self, filepath=None):
 
     def start_process(self, command, path=None):
         logfile, stderr = self.run_command("mktemp")
-        self.run_command("nohup %s >& %s &" % (command, logfile), path)
-
+        self.run_command("nohup %s > %s &" % (command, logfile), path)
+        # JK_Note: Change the nohup command from "nohup %s >& %s &" to "nohup %s > %s &"
```

Comment:
Why did you make this change?  With this, error messages will not be output to the log file.  I couldn't find a reference for my syntax, but the old syntax for this is `nohup command > logfile 2>&1 &`.